src/main/java/skinsrestorer/bukkit/skinfactory/UniversalSkinFactory.java

```diff
@@ -276,7 +279,12 @@ public void updateSkin(Player player) {
                     sendPacket(playerCon, addInfo);
                     sendPacket(playerCon, respawn);
 
-                    ReflectionUtil.invokeMethod(craftHandle, "updateAbilities");
+                    try {
+                        ReflectionUtil.invokeMethod(craftHandle, "updateAbilities");
+
+                    } catch (InvocationTargetException e) {
+                        e.getCause().printStackTrace();
+                    }
```

Comment:
```suggestion
                    ReflectionUtil.invokeMethod(craftHandle, "updateAbilities");
```
This should only be used for debugging purpose